src/main/resources/assets/lambdynlights/lang/de_de.json

```diff
@@ -4,16 +4,16 @@
   "lambdynlights.menu.canvas.1": "Mit Canvas ist es empfohlen, die dynamische Entitäten-Beleuchtung auszuschalten!",
   "lambdynlights.menu.canvas.2": "Dynamische Beleuchtung mit Canvas ist zur Zeit sehr Leistungsbedürftig!",
   "lambdynlights.option.block_entities": "Blockentitäten",
-  "lambdynlights.option.entities": "Entitäten",
+  "lambdynlights.option.entities": "Objekte",
   "lambdynlights.option.mode": "Dynamische Lichter",
-  "lambdynlights.option.water_sensitive": "Wasser-sensibel Prüfung",
+  "lambdynlights.option.water_sensitive": "Wasserempfindlichkeit",
   "lambdynlights.toast.first_time": "Automatisch ausgeschaltet, prüfen Sie die Mod- oder Videoeinstellungen nach!",
```

Comment:
1. That must either be:
`"prüfen Sie **in** den Mod- oder Videoeinstellungen nach!"`
or
`"prüfen Sie die Mod- oder Videoeinstellungen!"`
A mix doesn't make sense gramatically.

2. Default **is not** equal to automatic, it is rather `"Standardmäßig"`.